src/sidebar/oauth-auth.js

```diff
@@ -22,6 +22,11 @@ function auth($http, settings) {
    * @property {string} accessToken  - The access token itself.
    * @property {number} expiresIn    - The lifetime of the access token,
    *                                   in seconds.
+   * @property {Date} expiresAt      - The access token's expiry time
```

Comment:
This doesn't match the actual name of the property (`refreshAfter`) returned by the function.